src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
+  const headers = table.querySelectorAll('th');
```

Comment:
Similar to the previous comment, ensure that `table` is not `null` before attempting to query for headers. This will prevent potential runtime errors.